.travis/build_package.sh

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
The shebang `#!/bin/bash` has to be the first line. It cannot be the second line.